src/models/Booking.js

```diff
@@ -0,0 +1,12 @@
+import BookingQueries from '../services/booking/operations';
+
```

Comment:
This is not needed. Sequelize handles it.

Reply:
STALE & OUTDATED DATA.

Reply:
alright